bin/barclamp_lib.rb

```diff
@@ -529,8 +536,9 @@ def parse_extra_opt(opt, arg)
 
 def check_user_password
   if @username.nil? or @password.nil?
-    STDERR.puts "CROWBAR_KEY not set, will not be able to authenticate!"
-    STDERR.puts "Please set CROWBAR_KEY or use -U and -P"
+    STDERR.puts "Incomplete credentials, will not be able to authenticate!"
+    STDERR.puts "Please a crowbarrc configuration file, " \
```

Comment:
I feel the verb is missing here

Reply:
Thanks, as agreed, I'll fix in the next PR (to avoid retriggering CI here and move on the other PR needed on GMC1).